Consolidate and update Grafana docs

choldgraf opened the pull request:
This updates our `sops` and grafana access information to be up to date.

This is paired with https://github.com/2i2c-org/team-compass/pull/215

closes https://github.com/2i2c-org/pilot-hubs/issues/568

yuvipanda commented on docs/howto/operate/grafana.md:
```diff
@@ -1,7 +1,94 @@
+    grafana:
+      ingress:
+        hosts:
+          - GRAFANA_URL
```
```suggestion
          - <grafana-domain>
```

yuvipanda replied:
these take domains (no https:// prefix)

yuvipanda commented on docs/howto/operate/grafana.md:
```diff
@@ -1,7 +1,94 @@
+        tls:
+          - secretName: grafana-tls
+            hosts:
+              - GRAFANA_URL
```
```suggestion
              - <grafana-domain>
```

yuvipanda reviewed (changes requested):
Minor comment, LGTM otherwise.

choldgraf commented:
OK I've addressed @yuvipanda's comments, and while incorporating suggestions from @damianavila I tried to re-build the docs locally to get it working. In the process I discovered that there were several broken links in the pilot hubs docs, and also that there weren't clear instructions for installing your own docs environment. I've cleaned that up a bit in this PR as well, so we should now have guidelines for how to build the docs.

choldgraf commented on docs/howto/operate/new-hub/add-aws-hub.md:
```diff
@@ -14,123 +14,112 @@ discussions outlined in [#431](https://github.com/2i2c-org/pilot-hubs/issues/431
-1. Follow the instructions outlined in
-[Set up and use the the deployment scripts locally](./manual-deploy.md) to set up the
-local environment and prepare `sops` to encrypt and decrypt files.
+1. Follow the instructions outlined in [Set up and use the the deployment scripts locally](../manual-deploy.md) to set up the local environment and prepare `sops` to encrypt and decrypt files.
```
a lot of the text in this section wasn't properly formatted markdown, so the numbered lists etc weren't working. Most of the changes here are just adding in the proper whitespace so that this works

damianavila reviewed (approved):
Nice addendum about the local docs.
This one LGTM, @choldgraf.

choldgraf commented:
thanks all for the review :-)
